pkgs/test/nixpkgs-check-by-name/scripts/run-local.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2016
+
+set -euo pipefail
+
+cleanup_commands=()
+cleanup() {
+    echo -n >&2 "Cleaning up.. "
```

Comment:
Putting the redirect at the end of the command is more idiomatic. It doesn't belong as an argument.

Reply:
I kind of don't like that idiom, because it leads to `>&2` being all over the place, whereas putting it in front makes them all be fairly aligned. I guess I could declare a function like `debug() { echo >&2 "$@" }` to not have it all over the place.

Reply:
A `debug` function would be preferable. But I think idiomaticity is more important than aligning things vertically. Having `COMMAND OPTIONS PARAMETERS REDIRECT` or `REDIRECT COMMAND OPTIONS PARAMETERS` is cleaner than `COMMAND OPTIONS REDIRECT OPTIONS PARAMETERS`.

Reply:
Now introduced a `trace` (sounded better than `debug`, since it's not strictly for debugging) function for this purpose in https://github.com/NixOS/nixpkgs/pull/274591/commits/ca7201d88358394118120162f19bc75c7fec5a13